ckanext/switzerland/plugin.py

```diff
@@ -238,64 +240,27 @@ def _package_map_ckan_default_fields(self, pkg_dict):
                     resource["name"] = resource["title"]
         return pkg_dict
 
+    def _group_or_org_map_ckan_default_fields(self, group_or_org_dict):
+        if "title" in group_or_org_dict:
+            group_or_org_dict["display_name"] = group_or_org_dict["title"]
+        if "notes" in group_or_org_dict:
+            del group_or_org_dict["notes"]
+
+        return group_or_org_dict
+
     def _extract_lang_value(self, value, lang_code):
         new_value = sh.parse_json(value)
 
         if isinstance(new_value, dict):
             return sh.get_localized_value(new_value, lang_code, default_value="")
         return value
 
-    def _package_reduce_to_requested_language(self, pkg_dict, desired_lang_code):
+    def _reduce_to_requested_language(self, pkg_dict, desired_lang_code):
         # pkg fields
         for key, value in list(pkg_dict.items()):
```

Comment:
can you explain why we no longer need to pass the TypeError? I guess I am a little careful when we are working with our dicts that often contain unsuspected things/types

Reply:
or are you avoiding this occuring by ignoring the field in the next line if it is empty?

Reply:
Sorry, I'm not sure what you're commenting on--did this comment end up in the wrong place somehow?

Reply:
Ah, the placing of the comment is maybe a little confusing. The reduce-functions each had a try except block in them which is now left out.

Reply:
All of the `_reduce_*_language` methods have been completely deleted, because they are not being called in `_reduce_to_requested_language` any more. This is because we are not passing package dicts to that method any more, only group and organisation dicts, so they aren't useful. So it's not the case that only the TypeError part has been removed. ;)

Reply:
Okay, thank you so much for clarifying this to me! Approved 😀